src/pages/en/install/manual.md

```diff
@@ -146,7 +146,24 @@ If you want to include [UI framework components](/en/core-concepts/framework-com
 
 📚 Read Astro's [API configuration reference](/en/reference/configuration-reference/) for more information.
 
-## 6. Next Steps
+## 6. Create `tsconfig.json`
+
+Typescript is configured using `tsconfig.json`. Even if you don’t write TypeScript code, this file is important so that tools like Astro and VS Code know how to understand your project. Some features (like npm package imports) aren’t fully supported in the editor without a `tsconfig.json` file.
+
+Create `tsconfig.json` at the root of your project, and copy the code below into it:
+
+```json title="tsconfig.json"
+{
+  "extends": "astro/tsconfigs/base",
+  "compilerOptions": {
+    "types": ["astro/client"]
+  }
+}
+```
+
+📚 Read Astro's [TypeScript setup guide](en/guides/typescript/#setup) for more information.
```

Comment:
```suggestion
📚 Read Astro's [TypeScript setup guide](/en/guides/typescript/#setup) for more information.
```
Fixed link formatting, and will make a direct commit on this to make the link checker pass and ensure there aren't other errors to address